lib/http_throttle.dart

```diff
@@ -36,8 +36,9 @@ class ThrottleClient extends BaseClient {
       rethrow;
     }
 
-    var stream = response.stream
-        .transform(new StreamTransformer.fromHandlers(handleDone: (sink) {
+    var stream = response.stream.transform(
```

Comment:
not worth the extra dependency, but for the record this is a good use case for `tap` from `package:stream_transform`

```dart
var stream = response.stream.transform(tap((_) {}, onDone: resource.release));
```